### ISSUE TITLE
Isotope messing up float

### ISSUE THREAD

**Issue by cgrant992** [metafizzy/isotope#543]
I have a problem with a new project I'm working on using isotope.

I have various sizes for images (large, small, tall, wide) in a container with 930px width. For some reason, when I have the 'large' or 'wide' (they have the same widths) class set for the current item, if it's the very first item on the masonry list, it breaks the whole layout of all the other items.

If there's another item before the 'large' or 'wide' item, then the layout works fine. It's the same when I filter new categories as well. 

So basically, when I use a 'large' or 'wide' as the first column size, the layout breaks and items stop floating... anybody know what could cause this? Maybe isotope's having a hard time getting the correct container width for some reason? Or maybe my column math is off... but if that's the case, why would it work fine when another item 'small' is placed in front of it.

Any help would be greatly appreciated.
